openreview/conference/templates/liteConferenceWebfield.js

```diff
@@ -0,0 +1,249 @@
+var PUBLIC = false;
+
+var WILDCARD_INVITATION = CONFERENCE_ID + '/.*';
+var BUFFER = 0;  // deprecated
+var PAGE_SIZE = 50;
+
+var paperDisplayOptions = {
+  pdfLink: true,
+  replyCount: true,
+  showContents: true,
+  showTags: false
+};
+var commentDisplayOptions = {
+  pdfLink: false,
+  replyCount: true,
+  showContents: false,
+  showParent: true
+};
+
+// Main is the entry point to the webfield code and runs everything
+function main() {
+  if (args && args.referrer) {
+    OpenBanner.referrerLink(args.referrer);
+  } else if (PARENT_GROUP_ID.length){
+    OpenBanner.venueHomepageLink(PARENT_GROUP_ID);
+  }
+
+  Webfield.ui.setup('#group-container', CONFERENCE_ID);  // required
+
+  renderConferenceHeader();
+
+  renderSubmissionButton();
+
+  renderConferenceTabs();
+
+  load().then(renderContent).then(Webfield.ui.done);
+}
+
+function load() {
+  var notesP = $.Deferred().resolve([]);
+  var authorNotesP = $.Deferred().resolve([]);
+  var userGroupsP = $.Deferred().resolve([]);
+
+  if (PUBLIC) {
+    notesP = Webfield.api.getSubmissions(BLIND_SUBMISSION_ID, {
+      pageSize: PAGE_SIZE,
+      details: 'replyCount,invitation,original',
+      includeCount: true
+    });
+  }
+
+  if (user && !_.startsWith(user.id, 'guest_')) {
+    userGroupsP = Webfield.getAll('/groups', {
+      regex: CONFERENCE_ID + '/.*',
+      member: user.id,
+      web: true
+    });
+
+    authorNotesP = Webfield.api.getSubmissions(SUBMISSION_ID, {
+      pageSize: PAGE_SIZE,
+      'content.authorids': user.profile.id
+    });
+  }
+
+  return $.when(notesP, userGroupsP, authorNotesP);
+}
+
+// Render functions
+function renderConferenceHeader() {
+  Webfield.ui.venueHeader(HEADER);
+
+  Webfield.ui.spinner('#notes', { inline: true });
+}
+
+function renderSubmissionButton() {
+  Webfield.api.getSubmissionInvitation(SUBMISSION_ID, {deadlineBuffer: BUFFER})
+    .then(function(invitation) {
+      Webfield.ui.submissionButton(invitation, user, {
+        onNoteCreated: function() {
+          // Callback funtion to be run when a paper has successfully been submitted (required)
+          if (PUBLIC) {
+            promptMessage('Your submission is complete. Check your inbox for a confirmation email. ' +
+              'A list of all submissions will be available after the deadline.');
+          } else {
+            promptMessage('Your submission is complete. Check your inbox for a confirmation email. ' +
+              'The author console page for managing your submissions will be available soon.');
+          }
+
+          load().then(renderContent).then(function() {
+            $('.tabs-container a[href="#your-consoles"]').click();
+          });
+        }
+      });
+    });
+}
+
+function renderConferenceTabs() {
+  var sections = [{
+    heading: 'Your Consoles',
+    id: 'your-consoles',
+  }];
+
+  if (PUBLIC) {
+    sections.push({
+      heading: 'All Submissions',
+      id: 'all-submissions',
+    });
+  } else {
+    sections.push({
```

Comment:
I don't think we should show 'Your submissions' tab, the author has the Author console link to see them.

Reply:
Really? If we hide that the only thing they will see is a link to the Author Console, and some people won't even know what that is. I think it's kind of nice to be able to see the titles of the papers right on the conference homepage.

Reply:
this is no happening now in the live site. I'm worried they may think the submission is public because is listed in the home page.

Reply:
OK, I see. Well I can either remove it, or I can put a message at the top of that tab saying that these submissions aren't public.

Reply:
I wouldn't show it, same as the live site now: it currently shows: 'Console' and 'Recent Activity' tabs. Removing the 'Recent Activity' is enough.